### ISSUE TITLE
Release of SAGE's Graph Tokenizer Training code

### ISSUE THREAD

**Issue by scottjiao** [YangLing0818/VQGraph#4]
Dear authors, your recently released graph tokenizer training code only contains the teacher **GCN part**, while the reported metric in the paper is trained on **teacher SAGE**. Would you like to release the SAGE part, which could help reproduce the reported results?

**Comment by scottjiao** [comment]
Sage code released now
